README.md

```diff
@@ -8,11 +8,11 @@
 >These checklists emerged during the review and crafting processes as ways to avoid pitfalls, allowing protocol engineers to focus on more in-depth reviews.
 >Future workflow improvements will be reflected in these checklists.
 
-## Core Contracts
+## Checklists
 
-- [Standards](core/standards.md)
+### Core Contracts
 
-## Checklists
+- [Standards](core/standards.md)
```

Comment:
nit: the core contracts list isn't really a checklist, more a description/guidance

Reply:
yeah, but the repo is about checklists so it should go under checklists imo